src/modules/groupMethods.ts

```diff
@@ -210,7 +214,9 @@ export const groupMethods = {
             where: { groupId: id, archived: false },
             include: { group: true, user: true, roles: true },
         });
-        return memberships.map((m) => ({ ...m, group: addCalculatedGroupFields(m.group, sessionUser) }));
+        return Promise.all(
+            memberships.map(async (m) => ({ ...m, group: await addCalculatedGroupFields(m.group, sessionUser) })),
```

Comment:
isn't it the same groupId? why do we need to have editable flag on each membership?